tools/ncf.py

```diff
@@ -380,10 +380,11 @@ def parse_technique_methods(technique_file):
       if ifvarclass_context is not None:
         promise_class_context = class_context_and(class_context, ifvarclass_context)
 
-      if args:
-        res.append({'class_context': promise_class_context, 'promiser': promiser, 'method_name': method_name, 'args': args})
-      else:
-        res.append({'class_context': promise_class_context, 'promiser': promiser, 'method_name': method_name})
+      if not (method_name.startswith("_") or method_name.startswith("log")): 
```

Comment:
why?

Reply:
To prevent treating _class and log methods in the technique which fails in most case when reading the metadata of these methods, i needed it when na reports were in the technique file

Maybe we can remove that line